lib/oxidized/model/qtech.rb

```diff
@@ -0,0 +1,38 @@
+class QTECH < Oxidized::Model
+  comment '! '
+
+  cmd :secret do |cfg|
+    cfg.gsub! /^(snmp-server community(?: r[ow])?(?: \d)?) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(snmp-server user .+ auth \S+) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(username .+ password \d) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(enable password(?: level \d+)? \d) .+/, '\\1 <secret hidden>'
+    cfg
+  end
+
+  cmd 'show version' do |cfg|
+    cfg = cfg.each_line.to_a[1..-2]
+    comment cfg.reject { |line| line.match /^  (Copyright |All rights reserved$|Uptime is |Last reboot is )/ }.join
+  end
+
+  cmd 'show running-config' do |cfg|
+    cfg.lines.to_a[1..-2].join
```

Comment:
This should be equivalent to the helper function cfg.cut_both

Reply:
Done.